decimal.go

```diff
@@ -86,7 +89,7 @@ func stripTrailingDigits(s string, digits int) string {
 
 // formatNumber formats the number using base and decimals. For bases different
 // than 10, non-integer floating numbers are truncated.
-func formatNumber(ctx decimal.Context, n *decimal.Big, base, decimals int) string {
+func formatNumber(ctx decimal.Context, n *decimal.Big, base, decimals int, single bool) string {
```

Comment:
"single" doesn't make much sense inside the function. Maybe we should replace it inside the function with "raw", meaning "raw printing" (no formatting) or something similar. In fact, I think we should create two functions, "format" and "formatRaw" that would call this function with raw set to false and true, respectively. Then we could change the rest of the code to call the right function, instead of adding one more mysterious parameter to the call.